library/libraries.js

```diff
@@ -1971,5 +1971,39 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         }
         return false;
       }
+    },
+    'Svelte': {
+      id: 'svelte',
+      icon: 'svelte',
+      url: 'https://svelte.dev/',
+      npm: 'svelte',
+      test: function (win) {
+        // can be toggled with discloseVersion option
+        // see https://svelte.dev/docs/svelte-compiler#types-compileoptions
+        const versions = win.__svelte.v;
```

Comment:
@benmccann Do we need some try-catches or optional chaining here? `win.__svelte.v` is going to throw if someone's disabled `discloseVersion`, and we'll never get to the `document.querySelector('[class*="svelte-"]')` below.